### PR TITLE
lab5

### PR DESCRIPTION
**Pull request by KuhaarTar**

### CODE REVIEW

File: Header.h [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+	string nameSecond;
+private:
+
+	int healthFirst;
+	string damagePerAttackFirst;
+
+
+	int healthSecond;
+	string  damagePerAttackSecond;
+public:
```
Comment (ihar-b) [inline Header.h:19]: 2 Public poles change

File: Header.h [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+	string getName() {
+		return nameFirst;
+	}
+	int getHealth() {
+		return healthFirst;
+	}
+	string getDamagePerAttack() {
+		return damagePerAttackFirst;
+	}
+	/*---------*/
```
Comment (ihar-b) [inline Header.h:39]: delete useless comment

File: Header.h [hunk 3]
```diff
@@ -0,0 +1,92 @@
+using namespace std;
+#include <iostream>
+#include <string>
+#include <random>
```
Comment (ihar-b) [inline Header.h:4]: make backdown

**Review by KuhaarTar — COMMENTED** [review]
I change all of ur remarks

**Review by ihar-b — APPROVED** [review]
well done

File: Header.h [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+	}
+	void setHealth1(int h) {
+		healthSecond = h;
+	}
+	void setDamagePerAttack1(string d) {
+		damagePerAttackSecond = d;
+	}
+	string getName1() {
+		return nameSecond;
+	}
```
Comment (zveres-ss) [inline Header.h:51]: its really obvious when method getName1 returns nameSecond

**Pull request merged**